packages/pot/transfer_service.go

```diff
@@ -0,0 +1 @@
+package pot
```

Comment:
Start here by defining the new interface that will be used at the root of the function once you have the client id and the split